### PR TITLE
EVG-7782: Return 404 when test result DNE

### PR DESCRIPTION
**Pull request by julianedwards**

### CODE REVIEW

File: rest/test_results_routes.go [hunk 1]
```diff
@@ -15,7 +15,7 @@ import (
 
 ///////////////////////////////////////////////////////////////////////////////
 //
-// GET /testresults/task_id/{task_id}
+// GET /test_results/task_id/{task_id}
```
Comment (Kimchelly) [inline rest/test_results_routes.go:18]: Doesn't the REST service route also have to be changed?

**Review by Kimchelly — CHANGES_REQUESTED** [review]

File: rest/test_results_routes.go (hunk 1 above)
Comment (julianedwards) [reply rest/test_results_routes.go:18]: I am going to change it in another ticket, I don't want to break too many things right now.

Comment (Kimchelly) [reply rest/test_results_routes.go:18]: I thought nothing was using the test results routes right now. What's wrong with changing it?

Comment (julianedwards) [reply rest/test_results_routes.go]: Staging, but I will change it before I merge staging. It just requires me to also change timber and revendor and I am trying to debug something else before that.

**Review by Kimchelly — APPROVED** [review]

**Pull request merged**